module-1/lab-api-scavenger-game/your-code/challenge-2.ipynb

```diff
@@ -0,0 +1,610 @@
+{
```

Comment:
Good job!! Well done!! The code works because you are counting records otherwise you would have got undesired results. The same as in challenge 1, you should normalize the JSON object.